### DISCUSSION TITLE
articles/life-is-a-journey-2011

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#128] [Comments]
https://hutusi.com/articles/life-is-a-journey-2011 

 昨天的日历上，记录的是一首苏轼的词《临江仙·送钱穆父》：

**Comment by DeppWang** [comment]
在 2021 年尾巴上发现的 flomo、Notion、卡片笔记盒。全新的知识管理概念，赞

**Comment by adela-bytebase** [comment]
你好John，我是Bytebase的Adela，看到你star了我们的repo，感谢！想做个小调查，请问你这边是出于个人兴趣还是工作需求来关注我们项目的呢？

**Comment by hutusi** [comment]
> @DeppWang
> 在 2021 年尾巴上发现的 flomo、Notion、卡片笔记盒。全新的知识管理概念，赞

Notion用了一年，可以说是改变了自己的行为习惯；下半年开始认真用flomo，也是越用越顺心。

**Comment by hutusi** [comment]
> @adela-bytebase
> 你好John，我是Bytebase的Adela，看到你star了我们的repo，感谢！想做个小调查，请问你这边是出于个人兴趣还是工作需求来关注我们项目的呢？

个人兴趣呀，Bytebase做的不错，应该点赞~~

**Comment by DeppWang** [comment]
> > @DeppWang
> > 在 2021 年尾巴上发现的 flomo、Notion、卡片笔记盒。全新的知识管理概念，赞
> 
> Notion用了一年，可以说是改变了自己的行为习惯；下半年开始认真用flomo，也是越用越顺心。

卡片笔记，好的技巧加上刻意练习，很好的成长方式，赞

**Comment by maoruibin** [comment]
> 搞好家庭关系，把娃带好。
哈哈，太真实了 ~
